src/components/elements/accordion.tsx

```diff
@@ -161,15 +182,19 @@ const ItemExpanded = ({ is_default_open, child, child_idx, nodes, id }) => {
     )
 }
 
-ItemExpanded.propTypes = {
-    child: PropTypes.any,
-    child_idx: PropTypes.any,
-    id: PropTypes.string,
-    is_default_open: PropTypes.bool,
-    nodes: PropTypes.any,
+type SingleAccordionContentProps = {
+    children?: ChildType | ChildType[]
+    id?: string
+    is_default_open?: boolean
+    nodes?: ReactNode
```

Comment:
shouldn't it be an array of nodes? I looks like it's used as an array inside ItemExpanded 🙂
```suggestion
    nodes?: ReactNode[]
```